RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs

```diff
@@ -74,12 +74,24 @@ public enum ParTypeStatic
         /// </summary>
         Wind,
 
+        /// <summary>
+        /// Courage (Kled).
+        /// </summary>
         Courage,
 
+        /// <summary>
+        /// Crimson Rush (Vladimir).
+        /// </summary>
         CrimsonRush,
 
+        /// <summary>
+        /// Flow (Yasuo).
+        /// </summary>
         Flow,
 
+        /// <summary>
+        /// Fury (Renekton).
+        /// </summary>
         Fury,
```

Comment:
Should we specify that Yasuo's Wind is deprecated like:

```c#
/// <summary>
/// Wind (Yasuo before rework).
/// </summary>
``` 

Same for Renekton's rage?